crates/ln-dlc-node/src/ln_dlc_wallet.rs

```diff
@@ -123,9 +127,7 @@ impl Blockchain for LnDlcWallet {
 
     #[autometrics]
     fn get_network(&self) -> Result<Network, Error> {
-        self.ln_wallet
-            .network()
-            .map_err(|e| Error::BlockchainError(e.to_string()))
+        Ok(self.network)
```

Comment:
nit: This is now infallible, we could get rid of the result.

Reply:
You're right! I thought it was a trait method 🤦

Reply:
Ah, nvm. It _is_ a foreign trait method.